### ISSUE TITLE
Should we ignore ORCID provided with CrossRef DOI?

### ISSUE THREAD

**Issue by bryanwweber** [Niemeyer-Research-Group/ChemKED#10]
At [L183 of `validation.py`](https://github.com/Niemeyer-Research-Group/ChemKED/blob/master/chemked/validation.py#L183), we check the provided ORCID against the ORCID that is optionally provided from the CrossRef database. I would argue this check is unnecessary and will lead to false-warnings. We already validate the provided ORCID separately... if we want to suggest to users that they provide the ORCID of a paper author, we should do that in that validation function. That said, I don't think that we should provide this suggestion at all, since many more authors are not going to have ORCIDs than have them, unless we search the ORCID database and provide possible ORCIDs for the authors who are missing them. If we do provide the suggestion, it should be possible to turn it off as well. (Sorry I didn't catch this in the PR).

**Comment by kyleniemeyer** [comment]
I agree checking the DOI-based ORCID with provided ORCID isn't strictly necessary, but if the file doesn't provide ORCID (when there is one easily available from the reference DOI) then I *do* think we should suggest adding it in this location—mainly because I absolutely don't want to search for ORCID based on name.

I want to encourage including ORCID in the files, and since we basically get them "for free" with the DOI (occasionally, at least...) I'd like to keep this.

Actually, when we extend DOI support to DataCite DOIs ( #9 ), any such references will be more likely to have associated ORCIDs, since I think people using Zenodo and Figshare would be more likely to have those associated with their accounts (no reference to back this up).

**Comment by bryanwweber** [comment]
> then I do think we should suggest adding it in this location

Then it should be easy to turn this message off. Consider the warning messages in Cantera, and how annoying those are...

**Comment by kyleniemeyer** [comment]
OK, so I think the agreed to-do here is to allow disabling these warning messages.

I do still want to keep the check of provided ORCID vs. what the DOI returns, if only to give a correct value such that the file can be updated. The separate ORCID validation will not do that, just check that the given name matches what's in the system for the given ORCID.
